### PR TITLE
chore: always require curly braces

### PR DESCRIPTION
**Pull request by sheremet-va**
### Description

Made curly braces required as discussed with the team. Eslint doesn't support autofix for this, so I run prettier and eslint on top of it 👀 

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: packages/vitest/src/runtime/runners/test.ts [hunk 1]
```diff
@@ -118,26 +143,40 @@ export class VitestTestRunner implements VitestRunner {
       isExpectingAssertions,
       isExpectingAssertionsError,
       // @ts-expect-error _local is untyped
```
Comment (AriPerkkio) [inline packages/vitest/src/runtime/runners/test.ts:145]:
```suggestion
```

File: packages/vitest/src/runtime/runners/test.ts [hunk 2]
```diff
@@ -118,26 +143,40 @@ export class VitestTestRunner implements VitestRunner {
...
-    } = 'context' in test && test.context._local
-      ? test.context.expect.getState()
-      : getState((globalThis as any)[GLOBAL_EXPECT])
-    if (expectedAssertionsNumber !== null && assertionCalls !== expectedAssertionsNumber)
+    }
+      = 'context' in test && test.context._local
```
Comment (AriPerkkio) [inline packages/vitest/src/runtime/runners/test.ts:147]:
```suggestion
    }
      // @ts-expect-error _local is untyped
      = 'context' in test && test.context._local
```

**Comment by AriPerkkio** [comment]
Build is now passing but there's an actual test failure. Maybe some change added some unicode characters?

```
 × test/public-api.test.ts > passes down metadata when 'running in the browser' 488ms
     → Unexpected token '', "��z��" is not valid JSON
```

File: packages/vitest/src/node/cli/cli-config.ts [hunk 3]
```diff
@@ -602,24 +696,32 @@ export const cliOptionsConfig: VitestCLIOptions = {
...
+            description:
+              'Poll timeout in milliseconds for `expect.poll()` assertions (default: `1000`)',
             argument: '<timeout>',
           },
         },
         transform(value) {
-          if (typeof value !== 'object')
-            throw new Error(`Unexpected value for --expect.poll: ${value}. If you need to configure timeout, use --expect.poll.timeout=<timeout>`)
+          if (typeof value !== 'object') {
+            throw new TypeError(
```
Comment (AriPerkkio) [inline packages/vitest/src/node/cli/cli-config.ts:711]: Was the changed from `Error` to `TypeError` here intentional? Tests for this started to fail now.

Comment (sheremet-va) [reply packages/vitest/src/node/cli/cli-config.ts:711]: Not intentional, but sounds right. I think we have a eslint rule for that

**Comment by sheremet-va** [comment]
@AriPerkkio can you update coverage tests correctly? Since some lines have changed, snapshots _and_ tests themselves (after snapshot update) are now failing

**Comment by sheremet-va** [comment]
@AriPerkkio did you run the tests after snapshots? For me, it was not enough because there are tests that check hard-coded line numbers.

**Pull request merged**